### ISSUE TITLE
Reshape the tensor from [1, 3, 16, 256, 256] to [3, 16, 256, 256]

### ISSUE THREAD

**Issue by s922018** [NVIDIA-AI-IOT/torch2trt#737]
**Describe the bug**
I'm trying to reshape tensor like [1, 3, 16, 256, 256] to [3, 16, 256, 26] to do conv2d, and after training I try to convert the model by Trt.
I get this error message:
![image](https://user-images.githubusercontent.com/46773670/169225690-5aa1412b-cae8-45e6-bcd3-692dbfec1e2e.png)

I also use squeeze, but it still has same error. Is torch2trt not support to reshape tensor?

**Comment by JWLee89** [comment]
I think sharing the reproduction code (the model implementation, forward logic and `torch2trt` utilization) would help people provide useful suggestion / feedback. 
From the error message and description, it is difficult to follow exactly how `torch2trt` was utilized and the operations / order of operations inside of the model.

**Comment by s922018** [comment]
Input is a 3D tensor, so it's shape is (n, c, d, h, w). I want to reshape it to (n*d, c, h, w) and do 2Dconv.
Here is the code:

`
class TiedBlockConv2d(nn.Module):

    def __init__():
        ...
        self.out_planes = ..
        self.conv = nn.Conv2d(..)
        self.bn = nn.BatchNorm2d(..)

    def forward(self, x):
        n, c, d, h, w = x.size()
        x = x.permute(0, 2, 1, 3, 4)
        x = x.contiguous().view(n * d, c, h, w)

        x = self.conv(x)
        x = self.bn(x)
        x = self.relu(x)

        x = x.view(n, d, self.out_planes, h_o, w_o)

        x = x.permute(0, 2, 1, 3, 4)
        return x
`
